uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -226,6 +226,20 @@ define([
       };
 
       /**
+       * Checks if user can see file list in files tab. It needs to check for
+       * soft delete flag on leave request as well for files.
+       *
+       * @return {Boolean} true is user can see file list
+       */
+      this.canShowFileList = function () {
```

Comment:
I am not sure why do we need this feature? Why are we not showing the Files List on a certain condition? Please document the reason to add this in technical details.